server/pulp/server/db/model/__init__.py

```diff
@@ -700,3 +700,30 @@ class LazyCatalogEntry(Document):
     checksum = StringField()
     checksum_algorithm = StringField(regex=ALG_REGEX)
     data = DictField()
+
+
+class DeferredDownload(Document):
```

Comment:
Also every new model introduced that lives in platform must have ensure_indexes called in [`ensure_database_indexes()`](https://github.com/bmbouter/pulp/blob/e022f247f841800b0be18f7fcd0bdd5bb2f4ab3a/server/pulp/server/db/manage.py#L120)